test/10_isless.jl

```diff
@@ -58,7 +59,7 @@ module TestIsLess
     @test isless(v3, v2) === false
     @test isless(v3, v3) === false
 
-    # Check comparison with null
+    @testset "comparison with null" begin
```

Comment:
Indentation should be changed here, or maybe don't add a testset here (you didn't add it below for a similar code).